modules/containers_proto/include/hephaestus/containers_proto/bit_flag.h

```diff
@@ -0,0 +1,31 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include "hephaestus/containers/bit_flag.h"
+#include "hephaestus/containers/proto/bit_flag.pb.h"
+#include "hephaestus/serdes/protobuf/concepts.h"
+#include "hephaestus/serdes/protobuf/enums.h"
+
+namespace heph::serdes::protobuf {
+template <typename EnumT>
+struct ProtoAssociation<containers::BitFlag<EnumT>> {
+  using Type = containers::proto::BitFlag;
+};
+}  // namespace heph::serdes::protobuf
+
+namespace heph::containers {
+
+template <typename EnumT>
+auto toProto(proto::BitFlag& proto_bit_flag, const BitFlag<EnumT>& bit_flag) -> void {
+  proto_bit_flag.set_value(bit_flag.getUnderlyingValue());
```

Comment:
Don't we need a static_Cast here as well?

Reply:
It's not necessary, because casting from an int to an int of larger size is implicit. I could still add it, to make it clear a conversion is happening if you want.

Reply:
ah good point, also if clang-tidy doesn't complain I think we are good